step-core-model/src/main/java/step/plugins/alerting/rule/condition/AlertingRuleAction.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (C) 2020, exense GmbH
+ *
+ * This file is part of STEP
+ *
+ * STEP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * STEP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with STEP.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package step.plugins.alerting.rule.condition;
+
+public abstract class AlertingRuleAction {
```

Comment:
In the longterm all these classes should remain part of EE and registered to the package manager via plugin

Reply:
@jeromecomte 
I agree. I moved some abstract classes and interfaces from EE to OS just because the implementations of automation package and automation package manager are now in OS, so we need to have some common interfaces these (to be used in `YamlAutomationPackageSchemaGenerator` and for deserialziation).